src/app/health/page.tsx

```diff
@@ -13,8 +11,8 @@ export default function Health() {
         Health
       </Typography>
 
-      <Box width={'100%'}>
-        <EnhancedTable/>
+      <Box width={"100%"}>
```

Comment:
When we don't need things to be in JSX brackets we should just leave it as "" so width=""

Just a note for future code, don't spend all day removing brackets